internal/pkg/fakeroot/fakeroot.go

```diff
@@ -388,10 +393,10 @@ func IsUIDMapped(uid uint32) bool {
 		sylog.Fatalf("could not retrieve user with UID %d: %s", uid, err)
 	}
 	e, err := config.GetUserEntry(userinfo.Name)
-	if err != nil {
+	if err != nil && errors.Is(err, errNoMappingEntry) {
```

Comment:
Should it be `if err != nil && !errors.Is(err, errRangeTooLow)` ?

Reply:
Either way should work for now, but the question is what would happen if a new error case came up in the future.  It's a guess, but I think it is more likely that a new error case will more likely be one that should be ignored than included.  So I think the current check is the correct one.  This function is called "IsUIDMapped" sp all we want to check is whether or not it is mapped, which is what this error check is looking for.

Reply:
I think it's more explicit and safe to consider all errors and return false except for `errRangeTooLow` to avoid introduction of potential issues in the future

Reply:
I still disagree but not strongly enough to object further, so I made the change.